front/app/containers/Authentication/steps/AuthProviders/ClaveUnicaExpandedAuthProviderButton.tsx

```diff
@@ -61,11 +60,7 @@ const ClaveUnicaExpandedAuthProviderButton = memo<Props>(
     const [tacError, setTacError] = useState(false);
     const [privacyAccepted, setPrivacyAccepted] = useState(false);
     const [privacyError, setPrivacyError] = useState(false);
-    const isShowClaveUnicaConsentsEnabled = useFeatureFlag({
-      name: 'show_clave_unica_consents',
-    });
-    const showConsent =
-      showConsentOnFlow === flow && isShowClaveUnicaConsentsEnabled;
+    const showConsent = showConsentOnFlow === flow;
```

Comment:
`showConsentOnFlow` could just be a string literal here- it's never provided as a prop

Reply:
Should I remove the prop completely?